prototyping/MockArticle.py

```diff
@@ -0,0 +1,8 @@
+class Article:
```

Comment:
When in doubt just check google's style guide,  https://google.github.io/styleguide/pyguide.html#3163-file-naming. Afaik, there's no problem with having different file and class name. 
I, however did this out of pure convinience, since it seems that everyone is individually making their own article I named it this way to avoid merge conflicts in the future.